src/js/Controllers/UIController.js

```diff
@@ -147,6 +150,25 @@ class UIController {
                 }
 
                 return null
+            case "CancelInteraction":
+
+                const state2 = store.getState()
+                var app = state2.ui[state2.activeApp]
+                
+                if (rpc.params.functionID === 10 && app.isPerformingInteraction
+                     && (rpc.params.cancelID === undefined || rpc.params.cancelID === app.interactionCancelId)) {
+                    this.listener.send(RpcFactory.VRPerformInteractionAbortedResponse(app.interactionId - 1))
```

Comment:
Seems this is done elsewhere, the HMI should probably just respond to `VR.PerformInteraction` with `SUCCESS` immediately after receiving the request, since it doesn't actually do anything with the request.